build.sh

```diff
@@ -27,6 +27,7 @@ done
 my_hostname=`hostname`
 if [[ $my_hostname =~ "fair" ]]; then
   module purge
+  module load anaconda3/5.0.1
```

Comment:
This will end-up breaking the build on the fair cluster if you are using your own conda install.